pyasdf/asdf.py

```diff
@@ -406,6 +365,54 @@ def open(cls, fd, uri=None, mode='r',
 
         return self
 
+    @classmethod
+    def open(cls, fd, uri=None, mode='r',
+             validate_checksums=False,
+             extensions=None,
+             do_not_fill_defaults=False,
+             _get_yaml_content=False):
```

Comment:
Just wondering, that now that we have open as a front-end to _open_impl, does it need to accept the `_get_yaml_content` argument?  I get the impression that's just for internal use anyways.